controllers/external/util_test.go

```diff
@@ -175,7 +180,12 @@ func TestCloneTemplateResourceFound(t *testing.T) {
 		ClusterName: testClusterName,
 		OwnerRef:    owner.DeepCopy(),
 		Labels: map[string]string{
-			"test-label-1": "value-1",
```

Comment:
why are we dropping this label?

Reply:
It is covered by the precedence label below, which is a little more clearer with regards to whether precedence is correct, but also verifies that labels are applied in general. That's why I decided to drop it, as it doesn't seem necessary to have two labels. I can re-add it if you disagree.